numpy/fft/tests/test_pocketfft.py

```diff
@@ -258,6 +257,16 @@ def test_rfft(self):
                 np.fft.rfft(x, n=n) / n,
                 np.fft.rfft(x, n=n, norm="forward"), atol=1e-6)
 
+    def test_rfft_even(self):
+        x = np.arange(8)
+        y = np.fft.rfft(x, 4)
+        assert_allclose(y, [6.0, -2.0+2.0j, -2.0], rtol=1e-14)
```

Comment:
Could use the same sort of test as for the odd case, I like comparing to the full complex fft case.

Reply:
Done.